tests/acceptance/Makefile

```diff
@@ -0,0 +1,126 @@
+##=========================  Acceptance Tests =========================#
+include ./config.mk
+
+TAGNAME ?= default
+test-env-up:
+	@cd ../.. && docker build . -q -f ./tests/acceptance/shared/scripts/Dockerfile.build -t rke2-automated-${TAGNAME}
+
+# -d
+.PHONY: test-run
+test-run:
+	docker run  --name rke2-automated-test-${IMGNAME} -t \
+      -e AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}" \
+      -e AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}" \
+      -v ${ACCESS_KEY_LOCAL}:/go/src/github.com/rancher/rke2/tests/acceptance/modules/config/.ssh/aws_key.pem \
+       rke2-automated-${TAGNAME} sh -c 'cd ./tests/acceptance/entrypoint; \
```

Comment:
Could this be moved to a script for readability and maintainability?

Reply:
it will be on the next round !  i have added as next step on the PR